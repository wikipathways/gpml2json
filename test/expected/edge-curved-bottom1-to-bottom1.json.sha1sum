0e5d8e4985bca2f6af3c3a3d5dc05b35f860dac2 ?-
