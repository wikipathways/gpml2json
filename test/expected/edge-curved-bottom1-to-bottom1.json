{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e2bec","b531e","cfd74"],"drawAs":"rect","gpmlElementName":"Pathway","height":162.79148681896905,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":253.52002450009002,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/d4d7c0c00c334a84afe873034b124ad2ce07f7071a154a56131d556bddeda64e"},"entityMap":{"b531e":{"id":"b531e","textContent":"GeneProduct","wpType":"GeneProduct","x":143.52002450009002,"y":112.79148681896905,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32990,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"cfd74":{"id":"cfd74","textContent":"0014","wpType":"GeneProduct","x":26.19299253020759,"y":55.172900066173355,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32991,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e2bec":{"id":"e2bec","drawAs":"CurvedLine","borderWidth":1,"zIndex":32989,"points":[{"isAttachedTo":"cfd74","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":66.19299253020759,"y":75.17290006617336,"orientation":[0,1]},{"x":124.8565085151488,"y":152.79148681896905},{"isAttachedTo":"b531e","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":183.52002450009002,"y":132.79148681896905,"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["cfd74","b531e"]}}}
